apps/cli/src/commands/send/erc20.ts

```diff
@@ -55,6 +55,39 @@ export default class SendERC20 extends SendBaseCommand<typeof SendERC20> {
         };
     }
 
+    private async setAllowance(
+            publicClient: PublicClient,
+            walletClient: WalletClient,
+            amount: string,
+            tokenAddress: Address) {
+
+        const allowance = await publicClient.readContract({
+            address: tokenAddress,
+            abi: sunodoTokenAbi,  
+            functionName: "allowance",
+            args: [walletClient.account?.address as `0x${string}`, erc20PortalAddress],
```

Comment:
Instead of passing the walletClient as parameter, pass the owner address only.

Reply:
I still need the walletClient to writeContract and also need the account that's why i went for the object.